tests/lib/rule-tester/rule-tester.js

```diff
@@ -2107,11 +2175,12 @@ describe("RuleTester", () => {
                         invalid: [{
                             code: "var foo;",
                             errors: [{
+                                message: "Avoid using identifiers named 'foo'.",
                                 suggestions
                             }]
                         }]
                     });
-                }, "Error should have no suggestions on error with message: \"Avoid using identifiers named 'foo'.\"");
+                }, "Error should have suggestions on error with message: \"Avoid using identifiers named 'foo'.\"");
```

Comment:
This also seems wrong. The test case expects no suggestions but the actual lint message has suggestions, so the previous message "Error should have _no_ suggestions..." was correct.

"Error" here refers to the actual result (lint message produced by Linter). When the actual result doesn't match the expected result, it is assumed that the expected result is correct and the actual isn't (i.e., the test case is correct but the functionality being tested is not working as intended).